MAINTAINERS.md

```diff
@@ -23,37 +23,22 @@ matching the repository name:
 This guide is intended for *maintainers* — anybody with commit access to one or more laminas Framework
 repositories.
 
-We use the [git flow methodology](http://nvie.com/posts/a-successful-git-branching-model/) for
+We use [automatic relases](https://github.com/laminas/automatic-releases) for
 managing the various laminas Framework repositories. At a glance, this means:
 
-- a **master** branch. This branch MUST be releasable at all times. Commits and merges against
-  this branch MUST contain only bugfixes and/or security fixes. Maintenance releases are tagged
-  against master.
-- a **develop** branch. This branch contains *new features*, and will either become the next minor
-  (feature) release or next major release. Typically, major releases are reserved for backwards
-  *incompatible* changes, but can also be used to signal major new features.
+- two or more minor version branches of the current major release
+  - the highest minor version branch must not be released (i.e. does not have matching version tags like `1.0.0` 
+    for `1.0.x`).
+  - all other minor version branches must be released (i.e. have one or more version tags).
+- only unrelased branches may receive *new features*.
+- released branches must only receive bugfixes or security fixes.
+- releases (tags, next minor branch, github releases) will be performed automatically
+  by the automatic releases workflow. 
 
 Maintainers can choose to release new maintenance releases with each new patch, or accumulate
 patches until a significant number of changes are in place. Security fixes, however, must be
-released immediately and coordinated with the laminas Framework team to ensure an advisory is made, a
-CVE is created, and that the fix is backported to the Long Term Support release.
-
-> ## DO NOT PRESS THE GREEN BUTTON
->
-> ![Merge button](img/green-button.png)
->
-> GitHub provides a button, affectionately called the "green button," for merging pull requests from
-> the web interface.
->
-> **DO NOT EVER USE IT.**
->
-> Because we follow git flow, any changes made on `master` must also be merged to `develop`, and
-> this cannot be accomplished from the web interface. Additionally, we request that you annotate
-> the merge commits to reference the pull request and any associated issues; this practice
-> simplifies determining the source of a feature, and allows developers to discover the rationale
-> as well as any discussion that occurred.
->
-> The remainder of this document details how to merge pull requests to the repositories.
+released immediately and coordinated with the TSC to ensure an advisory is made, a
+CVE is created, and that the fix is backported to the Long Term Support releases.
```

Comment:
This part changes.

We don't have LTS anymore.

We have "security windows". Essentially, in the case of a security advisory and associated release(s), we make the patch to the earliest release branch we still support. From there, automatic-releases takes over, and does merge-up releases to each next latest release branch until it hits the latest. Each of those releases along the way will be listed in the security advisory.